x-pack/plugins/secops/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -100,30 +102,30 @@ export const AuthenticationTable = connect(
   }
 )(AuthenticationTableComponent);
 
-const getAuthenticationColumns = (startDate: number) => [
+const getAuthenticationColumns = (startDate: number): Array<Columns<AuthenticationsEdges>> => [
   {
     name: i18n.USER,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }: AuthenticationsEdges) => {
-      const userName = defaultToEmpty(node.user.name);
-      return (
-        <>
+    render: ({ node }) => {
+      const userName: string | null = get('user.name', node);
+      if (userName != null) {
+        return (
           <DraggableWrapper
             dataProvider={{
               and: [],
               enabled: true,
-              id: node._id,
-              name: userName!,
+              id: escapeDataProviderId(`authentications-table-${node._id}-user-${userName}`),
+              name: userName,
               excluded: false,
               kqlQuery: '',
               queryMatch: {
-                field: 'auditd.data.acct',
-                value: userName!,
+                field: 'user.name',
+                value: userName,
               },
               queryDate: {
                 from: startDate,
-                to: moment().valueOf(),
+                to: Date.now(),
```

Comment:
I've seen this a few other places (like in `HostsTable`), but what's the intent of using the current time for `queryDate.to`? Why would this not be the `to` value of the `GlobalTime` component that sets the time constraints for the widget itself?

Reply:
++

Reply:
It should probably be constrained to the global date time. The `Date.now()` kind of shows the bug more clearly it looks like.